### PR TITLE
Bug 2055729: change NodePerfCheck criteria to be based on average latency

### PR DESCRIPTION
**Pull request by rvanderp3**
TL;DR
The purpose of this PR is to change the alert criteria to take an average of the disk latency over the aggregate of the metric history set which stores up to approximately the last hour of metrics.  

The Problem
CI has uncovered a scenario where intermittent disk latency of over 100ms results in the `NodePerfCheck` test failing even though disk latency is not persistently poor for the node in question.  The intention of this change is to only fire the alert if there is persistent latency over 100ms.

Why is the alert being averaged over the period of an hour? 
The thought behind this is that if no problems are detected by the vSphere problem detector the check is only run, at most, every hour.  While high instantaneous latency events may be missed by averaging, a persistently poor performing disk will still cause this check to fail.  Alternatively, metric checks could be performed more frequently.  However, querying metrics is a rather expensive operation.  By taking the average of the last hour, we are getting a signal of a persistently poorly performing disk while minimizing the impact to vCenter.

### CODE REVIEW

**Comment by gnufied** [comment]
Could we have solve this in the place where we emit those alerts? i.e - average over larger time interval at alert rule rather than here? I do not mind doing it here as well, but just wanted to check.

**Comment by rvanderp3** [comment]
> Could we have solved this in the place where we emit those alerts? i.e - average over larger time interval at alert rule rather than here? I do not mind doing it here as well, but just wanted to check.

Perhaps, but we don't bubble up the actual latency value to the alert.  The thought was to do it here since we already have access to the available history.
